internal/app/cli/copy.go

```diff
@@ -94,7 +94,7 @@ func Copy(ctx context.Context, repo model.RepoSpec, toRepo model.RepoSpec, searc
 		}
 		if tmCopied { // copy tm name attachments if at least one tm has been copied successfully
```

Comment:
We should change the behavior here a bit, cause it feels strange for following szenario:

Given:   copy catalog A to a new catalog B
And Given:  a new attachment added to a TM in catalog A
When:  copy again catalog A to catalog B
Then:  the new attachment in catalog A is not copied to catalog B, as all TM's fail to copy with error "TM exists"

So at the moment it is only possible to get the new attachment copied with `--force`, but this means also that all other TM's and attachments in catalog B are overwritten.

For the scenario above, I would expect the new attachment can be copied without any `--force` and the output would look like this:
```
error   TM xy already exists as xy
error   could not import attachment  README.md: attachment already exists
OK      copied attachment new-attachment.json
```

Reply:
done